include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -78,47 +76,58 @@ class CBlitImageFilterBase : public impl::CSwizzleableAndDitherableFilterBase<Sw
 };
 
 // copy while filtering the input into the output, a rare filter where the input and output extents can be different, still works one mip level at a time
-template<typename Swizzle=DefaultSwizzle, typename Dither=CWhiteNoiseDither, typename Normalization=void, bool Clamp=true, class KernelX=CBoxImageFilterKernel, class KernelY=KernelX, class KernelZ=KernelX, typename lut_value_type = typename KernelX::value_type>
-class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Normalization,Clamp,KernelX,KernelX,KernelX>>, public CBlitImageFilterBase<typename KernelY::value_type,Swizzle,Dither,Normalization,Clamp>
+template<
+	typename Swizzle				= DefaultSwizzle,
+	typename Dither					= CWhiteNoiseDither,
+	typename Normalization			= void,
+	bool Clamp						= true,
+	typename BlitUtilities			= CBlitUtilities<>,
+	typename LutDataType			= float>
+class CBlitImageFilter :
+	public CImageFilter<CBlitImageFilter<Swizzle, Dither, Normalization, Clamp, BlitUtilities, LutDataType>>,
+	public CBlitImageFilterBase<Swizzle, Dither, Normalization, Clamp>
 {
 	public:
-		using blit_utils_t = CBlitUtilities<KernelX, KernelY, KernelZ>;
+		using blit_utils_t = BlitUtilities;
```

Comment:
stick a `static_assert(std::is_base_of_v<IBlitUtilities,blit_utils_t>,"Only template instantiations of `CBlitUtilities` are allowed as the `BlitUtilities` template argument!");` below

Reply:
Resolved.